src/sql/connection.py

```diff
@@ -328,20 +342,16 @@ def _close(cls, descriptor):
         if descriptor in cls.connections:
             cls.connections.pop(descriptor)
         else:
-            cls.connections.pop(str(conn.metadata.bind.url))
-
-        conn.session.close()
-
-    def close(self):
-        self.__class__._close(self)
+            cls.connections.pop(
+                str(conn.metadata.bind.url) if IS_SQLALCHEMY_ONE else str(conn.url)
+            )
+            conn.session.close()
 
-    @classmethod
-    def _get_curr_connection_info(cls):
+    def _get_curr_connection_info(self):
```

Comment:
did you rebase? `_get_curr_connection_info` was converted into a classmethod in the master branch, but here it looks like an instance method again